spring-boot-project/spring-boot-docker-compose/src/dockerTest/resources/org/springframework/boot/docker/compose/service/connection/db2/db2-compose.yaml

```diff
@@ -0,0 +1,18 @@
+services:
+  database:
+    image: '{imageName}'
+    ports:
+      - '50000'
+    privileged: true
+    environment:
+      - 'LICENSE=accept'
```

Comment:
What does this mean in practice?

Reply:
Which part?

Reply:
The `LICENSE=accept` bit.

Reply:
It's required by starting DB2 instance, testcontainers' `DB2Container` need this also. see https://github.com/testcontainers/testcontainers-java/blob/04206d9dff440e875906f01151f09ad04da3c68f/modules/db2/src/main/java/org/testcontainers/containers/Db2Container.java#L104

Reply:
I understand that it's required, I am asking what it means in practice that our project starts accepting that license.

Reply:
Not for production use, I guess.

Reply:
I don't think this is resolved as we haven't determined if we can legally accept the license for testing purposes. Knowing that Testcontainers has support for accepting the license is useful but it's opt-in and doesn't tell us if we can do the same automatically on CI and dev machines for testing purposes.

Reply:
It's not opt-in, Testcontainers accept the license by default.

Reply:
License is not accepted by default in Testcontainers. User must call `acceptLicense()`, which sets the env var, when defining `Db2Container`. See https://github.com/testcontainers/testcontainers-java/blob/main/modules/db2/src/test/java/org/testcontainers/junit/db2/SimpleDb2Test.java#L17

Reply:
My mistake, I forgot I added `container-license-acceptance.txt`.